default.yml

```diff
@@ -14,41 +12,12 @@ AllCops:
     - script/**/*
     - vendor/**/*
 
+#################### Layout ################################
+
 Layout/DotPosition:
   EnforcedStyle: trailing
 
-Lint/AmbiguousBlockAssociation:
-    Exclude:
-    - spec/**/*
-
-Metrics/BlockLength:
-  Exclude:
-  - config/routes.rb
-  - app/views/api/**/*
-
-Metrics/LineLength:
-  Max: 110
-  # To make it possible to copy or click on URIs in the code, we allow lines
-  # contaning a URI to be longer than Max.
-  AllowURI: true
-  URISchemes:
-    - http
-    - https
-  Exclude:
-    - config/routes.rb # Annotations run long.
-
-Naming/FileName:
-  Enabled: false
-
-Rails/HasAndBelongsToMany:
-  Enabled: false
-
-Rails/SkipsModelValidations:
-  Exclude:
-    - spec/**/*
-
-Style/CollectionMethods:
```

Comment:
Did you intentionally drop this one or was that an oversight?

Reply:
Whoops, thanks for catching that. Added it back.